compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -94,9 +94,11 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
         def showExtractors: String =
           Extractors.showTree(using QuotesImpl.this)(self)
         def show: String =
-          SourceCode.showTree(using QuotesImpl.this)(self)(SyntaxHighlight.plain)
+          SourceCode.showTree(using QuotesImpl.this)(self)(SyntaxHighlight.plain, fullNames = true)
+        def showShortTypes: String =
```

Comment:
What do you think about the additional `showShortTypes` @nicolasstucki ? Otherwise we will always need to bring in the compiler in with `SyntaxHighlight`

Reply:
I would call it `showShort`.

Reply:
Changed and added it to interfaces.